apps/server/src/controllers/plant.controller.js

```diff
@@ -60,10 +60,22 @@ const deletePlantById = async (req, res) => {
     }
 }
 
+const getPlantById = async (req, res) => {
+    try {
+        const plantId = req.params.plantId;
```

Comment:
actually you need to validate if it's a valid plant id and not an vulnerable injection  value...

Reply:
> actually you need to validate if it's a valid plant id and not an vulnerable injection value..

understand. security jej